src/generated/serverdata/v1/serverdata.ts

```diff
@@ -0,0 +1,98 @@
+/**
```

Comment:
should we ignore these files as they are autogenerated?  argos already has a build hook that calls your script and it is probably best practice so nobody uses out of dat versions of these files.

Reply:
sure i can ignore them